apps/api/src/app/content-templates/usecases/compile-template/compile-template.spec.ts

```diff
@@ -61,10 +62,29 @@ describe('Compile Template', function () {
       })
     );
 
+    it('should allow the user to specify handlebars helpers'),
+      async function () {
+        const result = await useCase.execute(
+          CompileTemplateCommand.create({
+            templateId: 'custom',
+            data: {
+              branding: {
+                color: '#e7e7e7e9',
+              },
+              message: 'hello world',
+            },
+            customTemplate: '<div>{{titlecase message}}</div>',
+          })
+        );
+      };
+
```

Comment:
There seems to be a closing parenthesis problem here. The async function should be a second parameter . 

`
it('test name', async function () {
..content of test..
});
`